src/content/guides/creating-clusters-via-crs-on-azure/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Creating tenant clusters on Azure via Control Plane Kubernetes API
+description: This guide will walk you through the process of tenant cluster creation via Control Plane Kubernetes on Azure.
+date: 2020-09-23
+type: page
+weight: 100
+tags: ["tutorial"]
+---
+
+## How does cluster creation work
+
+Starting from version {{% first_azure_nodepools_version %}} on Azure, Giant Swarm introduced a feature to create multiple [node pools](https://docs.giantswarm.io/basics/nodepools/) on Azure.
+Alongside node pools support, a new API version for cluster management was released.
+
+All the tenant clusters, created with release version {{% first_azure_nodepools_version %}} and newer, are managed as [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in the Control Plane.
+
+At a high-level, the Control Plane API is used to manage the following CRs:
+
+- [Cluster](/reference/cp-k8s-api/clusters.cluster.x-k8s.io/) - represents a Kubernetes cluster excluding worker nodes.
+- [MachinePool](/reference/cp-k8s-api/machinepools.exp.cluster.x-k8s.io/) - represents a node pool.
+
+The CRs above then reference provider specific implementations. In our case, for clusters on Azure, they are:
+
+- [AzureCluster](/reference/cp-k8s-api/azureclusters.infrastructure.cluster.x-k8s.io/) - represents a tenant cluster.
```

Comment:
Should there also be a mention of `AzureMachine` for tenant cluster masters?

Reply:
It's not referenced by `Cluster` or `MachinePool`. But I'll add it down below.